### PR TITLE
feat: ⚙ Removed underline on Explore button(#28)

### PR DESCRIPTION
**Pull request by kkhitesh**
## Related Issue 
Fixes issue #28 

- Information about the related issue 

Closes: # 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h1 class="my-4 text-5xl font-bold leading-tight">
             An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀
           </p>
           <a href="https://github.com/web3community/start-here" target="_blank" rel="noopener">
-            <button style="background: #0086D6" class="mx-auto lg:mx-0 hover:underline text-white rounded-md my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
```
Comment (mkubdev) [inline index.html:89]: Hi, you should remove the whole class hover:underline :)

**Review by mkubdev — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kkhitesh) [reply index.html:89]: Made the requested changes, thanks

**Review by shuklaritvik06 — APPROVED** [review]
LGTM :)

**Review by vinzvinci — APPROVED** [review]
LGTM 👍

**Pull request merged**